truss/remote/baseten/api.py

```diff
@@ -130,19 +131,18 @@ def create_model_from_truss(
         s3_key: str,
         config: str,
         semver_bump: str,
-        client_version: str,
         allow_truss_download: bool = True,
         deployment_name: Optional[str] = None,
         origin: Optional[b10_types.ModelOrigin] = None,
     ):
         query_string = f"""
-            mutation {{
+            mutation ($trussUserEnv: String) {{
```

Comment:
Wondering if we should add this as part of the request headers in `_post_graphql_query` so we don't have to specify it in every GQL query. That kind of consistency may be good for telemetry as well.

Would also be nice to centrally process this info in the backend through a middleware - not sure if it's feasible though.